### ISSUE TITLE
Support svg makeProps

### ISSUE THREAD

**Issue by davesnx** [davesnx/styled-ppx#141]

**Comment by davesnx** [comment]
We supported most svg props, what we lacked was accessibility properties. Added here https://github.com/davesnx/styled-ppx/pull/149
